workers/unity/Packages/com.improbable.gdk.buildsystem/Configuration/SpatialOSBuildConfiguration.cs

```diff
@@ -8,18 +8,18 @@ namespace Improbable.Gdk.BuildSystem.Configuration
 {
     [CreateAssetMenu(fileName = "SpatialOS Build Configuration", menuName = EditorConfig.BuildConfigurationMenu)]
     public class SpatialOSBuildConfiguration : SingletonScriptableObject<SpatialOSBuildConfiguration>
-    {        
-        [SerializeField] public List<WorkerBuildConfiguration> WorkerBuildConfigurations = 
+    {
+        [SerializeField] public List<WorkerBuildConfiguration> WorkerBuildConfigurations =
             new List<WorkerBuildConfiguration>();
-        
+
         [SerializeField] private bool isInitialised;
-        
+
         public BuildEnvironmentConfig GetEnvironmentConfigForWorker(string workerType, BuildEnvironment environment)
         {
             var config = WorkerBuildConfigurations.FirstOrDefault(x => x.WorkerType == workerType);
             if (config == null)
             {
-                Debug.LogWarning($"The worker type {workerType} does not have a build configuration.");
+                Debug.LogWarning($"The worker type {workerType} does not have a build configuration.", this);
```

Comment:
Is passing `this` as a context object useful for a user? Given they need to edit an instance of the scriptable object? 

Maybe give a path to the `.asset` file? (If that can be done)

Reply:
Very useful :) Clicking on the log highlights the asset!

![context](https://user-images.githubusercontent.com/31209570/46946917-2a0a4c80-d071-11e8-9b8d-03c79407fed3.gif)

Reply:
Ah nice its smart enough to figure it out :